src/components/Vqb.vue

```diff
@@ -61,6 +61,9 @@ export default class Vqb extends Vue {
 
   openStepForm(params: PipelineStep, index: number) {
     // after that, we delete from params to pass down the others keys to initialValue
+    if (this.isEditingStep) {
```

Comment:
Please, add a comment here in the code explaining why toggling step is required if an edition is ongoing